lib/fileSystemAssetLoader.js

```diff
@@ -0,0 +1,99 @@
+// This is a clone of
+// https://github.com/percy/percy-webdriverio/blob/master/src/fileSystemAssetLoader.js
+
+const fs = require('fs');
```

Comment:
`const { promises: fs }  = require('fs') ` and drop the `Sync` usages?

Reply:
just cloning the upstream here, can definitely be improved.